app/cdap/components/hydrator/components/TopPanel/ActionButtons/index.tsx

```diff
@@ -252,6 +266,17 @@ export const ActionButtons = ({
               <ButtonLabel>Import</ButtonLabel>
             </div>
           </CommonButton>
+          <label htmlFor="pipeline-import-config-link">
+            {/* The onClick here is to clear the file, so if the user uploads the same file
+            twice then we can show the error, instead of showing nothing */}
+            <HiddenInput
```

Comment:
so this hidden input was necessary?

Reply:
yes, basically it plays the same functionality as the one in angular